The plugin should print an excplicit error message if the maven version is lower than the 3.2.5

There's too many confused users reporting guice errors raised because the new components cannot be resolved. I think returning an error message could clear the confusion a bit.

> If it does, how come there are so many users running an older version of maven and reporting errors only now once they upgraded to 2.14.0?

I don't know. They lie?

apache-maven-3.0.5:
```
[ERROR] Failed to execute goal org.codehaus.mojo:versions-maven-plugin:2.12.0:help (default-cli) on project standalone-pom: The plugin org.codehaus.mojo:versions-maven-plugin:2.12.0 requires Maven version 3.2.5 -> [Help 1]
```

```
[INFO] --- versions-maven-plugin:2.14.0:help (default-cli) @ standalone-pom ---
[INFO] Versions Maven Plugin 2.14.0
  The Versions Maven Plugin is used when you want to manage the versions of
  artifacts in a project's POM.

This plugin has 35 goals:

versions:commit
```

This shows - that prerequisite IS BEING checked by maven, even so old as 3.0.5, as long as plugin declares it.
2.14.0 doesn't - and `:help` is executing fine (as doesn't need much more from maven 3.2.5).

--- a/versions-maven-plugin/pom.xml
+++ b/versions-maven-plugin/pom.xml
@@ -15,6 +15,10 @@
   <name>Versions Maven Plugin</name>
   <description>The Versions Maven Plugin is used when you want to manage the versions of artifacts in a project's POM.</description>
 
+  <prerequisites>
+    <maven>${mavenVersion}</maven>
+  </prerequisites>
+
   <dependencies>
     <dependency>
       <groupId>org.codehaus.mojo.versions</groupId>

